index.d.ts

```diff
@@ -1108,6 +1108,8 @@ type Execa$<OptionsType extends CommonOptions = {}> = {
 
 	Cannot use the following options: `all`, `cleanup`, `buffer`, `detached`, `serialization`, `signal` and `lines`. Also, the `stdin`, `stdout`, `stderr`, `stdio` and `input` options cannot be an array, an iterable or a web stream. Node.js streams must have a file descriptor unless the `input` option is used.
 
+	Unlike $\`command\`, the `stdin` option defaults to `"inherit"`, not `["pipe", "inherit"]`.
```

Comment:
`child_process.spawnSync()` does not return any child process. So we cannot do the manual piping logic we use to support `["pipe", "inherit"]`.